uploadFlow.sh

```diff
@@ -21,5 +21,19 @@ if [ -z $FLOW_TOKEN ] ;
 then
 	return 1
 else
-	curl $CURL_ARGS -X POST -H "flow-token: $FLOW_TOKEN" -H "Content-Type: application/json" "$HOST/repository/flows" --data-binary "@src/main/flows/$FLOW.json"
+	http_response=$(curl $CURL_ARGS -s -o uploadFlowResponse.txt -w "%{http_code}" -X POST -H "flow-token: $FLOW_TOKEN" -H "Content-Type: application/json" "$HOST/repository/flows" --data-binary "@src/main/flows/$FLOW.json")
 fi
+
+if [ $http_response != "200" ];
+then
+  if [ $http_response == "302" ];
+  then
+    echo "Got unexpected HTTP response ${http_response}. This is likely due to your token being incorrect."
+  else
+    echo "Got unexpected HTTP response ${http_response}. This is likely an error."
+  fi
+else
+  cat uploadFlowResponse.txt
+fi
+
+rm uploadFlowResponse.txt
```

Comment:
To avoid getting "No such file or directory" if the curl command on line 24 does not write the output file:

```suggestion
[ -e uploadFlowResponse.txt ] && rm uploadFlowResponse.txt
```

This approach could also potentially be applied to all scripts in this PR.

Reply:
Thanks Tim, now added!